.github/actions/do_build_ock/action.yml

```diff
@@ -164,8 +164,8 @@ runs:
               -DCA_CL_DISABLE_UNITCL_VECZ_CHECKS=${{ inputs.disable_unitcl_vecz_checks }}
               -DCA_GTEST_LAUNCHER="${{ inputs.gtest_launcher }}"
               -DCA_BUILD_32_BITS=${{ inputs.build_32_bit }}
-              -DCMAKE_C_COMPILER=${{ inputs.c_compiler }}
-              -DCMAKE_CXX_COMPILER=${{ inputs.cxx_compiler }}
+              -DCMAKE_TOOLCHAIN_FILE=${{ inputs.toolchain_file }}
+              -DCMAKE_CROSSCOMPILING=${{ inputs.cross_compiling }}
```

Comment:
`CMAKE_CROSSCOMPILING` should not need to be set manually. I understand we used to do that in our internal builds, but we don't need to replicate that here, CMake will set that automatically if the toolchain file specifies a different system name than the host system name.

Reply:
Support for CMAKE_CROSSCOMPILING has been removed.